Dockerfile-go

```diff
@@ -0,0 +1,69 @@
+FROM golang:1.9.3-alpine as build
```

Comment:
why is this dockerfile at the root of the repo, instead of in the lib/go subtree?